### DISCUSSION TITLE
Holidays in Belgium

### DISCUSSION THREAD

**Discussion by rlarroye** [openpotato/openholidaysapi.website#33] [Q&A]
Hello,

Thank you for your very helpful API.

For school holidays in Belgium vacations for French speakers (code: “FR”). The vacations displayed are not the correct ones, as there are excess “nationwide” vacations that are sent back.

For example I do the request : https://openholidaysapi.org/SchoolHolidays?countryIsoCode=BE&languageIsoCode=EN&validFrom=2025-04-01&validTo=2025-05-30&subdivisionCode=BE-FR

The response is : 

`[
  {
    "id": "93ea7c9c-c3e4-4384-9c46-7cd3567f34f4",
    "startDate": "2025-04-07",
    "endDate": "2025-04-21",
    "type": "School",
    "name": [
      {
        "language": "EN",
        "text": "Spring Holidays"
      }
    ],
    "nationwide": true
  },
  {
    "id": "374b8c60-0cad-417f-9107-da66c3631573",
    "startDate": "2025-04-28",
    "endDate": "2025-05-11",
    "type": "School",
    "name": [
      {
        "language": "EN",
        "text": "Spring Holidays"
      }
    ],
    "nationwide": false,
    "subdivisions": [
      {
        "code": "BE-FR",
        "shortName": "FR"
      }
    ]
  }
]`

Instead of : 

`
[
  {
    "id": "374b8c60-0cad-417f-9107-da66c3631573",
    "startDate": "2025-04-28",
    "endDate": "2025-05-11",
    "type": "School",
    "name": [
      {
        "language": "EN",
        "text": "Spring Holidays"
      }
    ],
    "nationwide": false,
    "subdivisions": [
      {
        "code": "BE-FR",
        "shortName": "FR"
      }
    ]
  }
]`

We've made a filter to remove the excess ones.

Do you know if we can modify the code to remove nationwide school holidays?

Thank you

**Comment by fstueber** [comment]
Hello, thanks for the feedback. This is by design. Asking for BE-FR means: Give me all vschool holidays for this subdivision (also nationwide, of course). As you mentioned, you can easily filter the result.

**Comment by fstueber** [comment]
And just in case: Raw data for Belgium you can find here: https://github.com/openpotato/openholidaysapi.data/tree/develop/src/be/holidays
